### PR TITLE
Adding power procedure

### PR DESCRIPTION
**Pull request by chris-milsted**

### CODE REVIEW

**Comment by cannischan** [comment]
the name of the md...**shutdown** instead of shutown?

File: docs/operations/shutown-procedure.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+## Scale down application workloads to stop disk IO
+
+First we want to stop disk I/O workloads gracefully and flush all the data to disk. To do this follow your standard kubernetes workflow or run book for scaling workloads to zero, it will probably look something similar to:
+
+```bash
+kubectl scale statefulset,deployment -n mynamespace --all --replicas=0
+```
+
+At this point, there should be nothing running that is generating I/O to any Ondat volumes and we are ready to stop Ondat. We need to stop Ondat before powering down kubernetes, as Ondat is designed to create new replicas of data for volumes that are no longer available. We do not want Ondat to do it's normal recovery actions as soon as we start shuting down nodes as this is a planned event. By scaling down the workloads and shutting down Ondat we will freeze all of the data in place and maintain the consistency.
```
Comment (cannischan) [inline docs/operations/shutown-procedure.md:36]: to do **its** normal...

**Comment by cannischan** [comment]
I would argue that we can remove all the 'please' in the text so we keep the writing style more or less consistent with other entries, feel free to suggest otherwise

**Comment by hubvu** [comment]
@cannischan @chris-milsted I will be submitting layout changes and the suggestion from Cannis soon.

File: docs/operations/shutown-procedure.md (hunk 1 above)
Comment (hubvu) [reply docs/operations/shutown-procedure.md]: fixed

**Comment by hubvu** [comment]
> I would argue that we can remove all the 'please' in the text so we keep the writing style more or less consistent with other entries, feel free to suggest otherwise

fixed.

**Pull request merged**